devops/scripts/deploy-public-upstream.sh

```diff
@@ -41,7 +41,7 @@ for commit in $newcommits; do
     delete=(${commit})
     for remove in "${delete[@]}"; do
       for i in "${commits[@]}"; do
-        if [ [ ${commits[i]} = $remove ]]; then
+        if [ [ ${commits[$i]} = $remove ]]; then
```

Comment:
i fix lgtm.
```suggestion
        if [[ ${commits[$i]} == "$remove" ]]; then
```
If you're going to use bash [[, go ahead and double equal it. I'm not sure about all the quotes on $commits so left it alone but technically shellcheck would want that quoted too.